dns/zone-configs/k8s.io._0_base.yaml

```
@@ -264,6 +264,10 @@ release-0-2.cluster-api.sigs:
 master.cluster-api.sigs:
   type: CNAME
   value: master--kubernetes-sigs-cluster-api.netlify.app.
+# https://github.com/kubernetes-sigs/cluster-api-provider-azure docs (@CecileRobertMichon, @nader-ziada, @devigned)
+capz.sigs:
+  type: CNAME
+  value: kubernetes-sigs-capz.netlify.app.
```

```suggestion
  value: kubernetes-sigs-cluster-api-provider-azure.netlify.app.
```

this still needs to be kubernetes-sigs-cluster-api-provider-azure.netlify.app because that's where the site is currently (https://github.com/kubernetes/org/issues/2122#issuecomment-681058974)

---

oh ok